src/QMCHamiltonians/NonLocalECPComponent.cpp

```diff
@@ -140,11 +140,7 @@ NonLocalECPComponent::RealType NonLocalECPComponent::evaluateOne(ParticleSet& W,
     {
       deltaV[j] = r * rrotsgrid_m[j] - dr;
       W.makeMove(iel, deltaV[j]);
-#if defined(QMC_COMPLEX)
-      psiratio[j] = psi.ratio(W, iel) * sgridweight_m[j] * std::cos(psi.getPhaseDiff());
-#else
-      psiratio[j] = psi.ratio(W, iel) * sgridweight_m[j];
-#endif
+      psiratio[j] = psi.full_ratio(W, iel) * sgridweight_m[j];
```

Comment:
Rename the full_ratio function to evaluateFullRatio in TrialWaveFunction

Reply:
Renamed.